app/services/data_migrations/backfill_work_history_status.rb

```diff
@@ -0,0 +1,15 @@
+module DataMigrations
+  class BackfillWorkHistoryStatus
+    TIMESTAMP = 20210930132233
+    MANUAL_RUN = false
+
+    def change
+      ApplicationForm
+      .joins(:application_work_experiences)
+      .where(recruitment_cycle_year: 2022, work_history_status: nil)
+      .where.not(application_work_experiences: { id: nil })
```

Comment:
I don't think you need this line because the `INNER JOIN` above will stop this query returning any application forms that have no work experiences.

Reply:
Good shout